PowerEditor/src/NppIO.cpp

```diff
@@ -626,7 +629,7 @@ void Notepad_plus::doClose(BufferID id, int whichOne, bool doDeleteBackup)
 		}
 
 		if (PathFileExists(buf->getFullPathName()))
-			_lastRecentFileList.add(buf->getFullPathName());
+			fileFullPath += buf->getFullPathName();

```

Comment:
true, but it's rather a sementic issue: 
`+=` is a concatanate operation, that's make us think there's already a string in it and we want to keep it.
So yes, please change it to make the intention more clear.